docs/fastdds/use_cases/large_data/large_data.rst

```diff
@@ -139,6 +139,41 @@ For receiving sockets, the command is:
 
 .. _flow-controllers:
 
+Increasing the Transmit Queue Length of an interface
+----------------------------------------------------
+The Transmit Queue Length (``txqueuelen``) is a TCP/UDP/IP stack network interface value that sets the number of packets allowed per kernel transmit queue of a network interface device. By default, the ``txqueuelen`` value for Ethernet interfaces is set to ``1000`` in Linux. This value is adequate for most Gigabit network devices. However, in some specific cases, the txqueuelen setting should be increased to avoid overflows that drop packets. Similarly, choosing a value that is too large can cause added overhead resulting in higher network latencies.
```

Comment:
Please insert line breaks after each sentence. This is part of our style guidelines for documentation.

Also ensure that lines have less than 120 characters.